### ISSUE TITLE
SMS Opt In Path configuration for CGG

### ISSUE THREAD

**Issue by DFurnes** [DoSomething/voting-app#451]
We need to fill in the opt-in paths for SMS messaging on Celebs Gone Good:

```
MOBILE_OPT_IN_PATH_US = xxxxxxx
MOBILE_OPT_IN_PATH_BR = xxxxxxx
MOBILE_OPT_IN_PATH_MX = xxxxxxx
```

/cc @DeeZone @classicfreddie

**Comment by classicfreddie** [comment]
MOBILE_OPT_IN_PATH_US = 196577

Will have BR/MX today.

**Comment by DFurnes** [comment]
@classicfreddie Thanks! I just set that on Cats Gone Good and CGG.

/cc @DeeZone

**Comment by DFurnes** [comment]
@classicfreddie Hey Freddie, any updates on the `BR` and `MX` opt-in paths?

**Comment by classicfreddie** [comment]
@DFurnes still waiting on mGage to help us figure out what IDs to point to, I'll send updates on that ASAP.

**Comment by DFurnes** [comment]
No prob, thanks @classicfreddie!

**Comment by classicfreddie** [comment]
For US messaging, I tested with the phone number 702-439-0195, but I'm not receiving messaging or being added into the Mobile Commons opt in path. Let me know if you want me to open this as a new issue @DFurnes

**Comment by DFurnes** [comment]
@classicfreddie If you can make that as a new issue, that'd be great! Thanks!

**Comment by deezone** [comment]
@DFurnes Here's the SMS values for CGG2015. There's differences in the values that need to be sent to our SMS providers based on the `user_country`.
- Mobile Commons
  - `mobile_opt_in_path_id => "196577"`
- mGage (Mexico)
  - `mobile_opt_in_path_id => "86865"`
  - `mobile_opt_in_keyword => "MXCGG"`
- mGage (Brazil)
  - `mobile_opt_in_path_id => "38383"` (to be updated once the shortcode is approved in Brazil)
  - `mobile_opt_in_keyword => "BZCGG"`

cc: @classicfreddie

**Comment by deezone** [comment]
@DFurnes I don't think the `mobile_opt_in_keyword` was implemented? Currently the values are hard coded in the related consumer.

**Comment by DFurnes** [comment]
@DeeZone Correct. I'll create an issue for adding those to the payload.

Closing this.
